lessons/cryptography/Encoding/encoding.md

```diff
@@ -0,0 +1,87 @@
+# [Cryptography] Encoding
+*Learn about encoding, a way to reformat information.*
+
+![banner alt text](.rsrc/banner.png)
+
+# Objectives
+- ASCII
+- Unicode
+- Base64
+
+# Introduction
+Encoding refers to converting information into a specific form. For example, character encoding refers to converting text (which is a sequence of characters) into binary representation (0's and 1's) so that a computer can understand the text. Characters include letters and symbols (such as !, @, #, and more). The most common types of encoding include ASCII, unicode, and base64. A tool called [CyberChef](https://gchq.github.io/CyberChef/) can be useful for encoding and decoding messages. 
```

Comment:
This is not inherently true. Base64 which is character encoding is not binary data.